screens/auth/Signup.tsx

```diff
@@ -19,6 +19,17 @@ const styles = StyleSheet.create({
     alignItems: 'center',
     paddingTop: 60,
   },
+  button: {
+    backgroundColor: '#8790E0',
+    shadowColor: '#8790E0',
+    shadowOffset: {
+      width: 0,
+      height: 9,
+    },
+    shadowOpacity: 0.2,
+    shadowRadius: 11.95,
```

Comment:
![rick-and-morty-precision](https://user-images.githubusercontent.com/19322697/160589867-9db591be-dded-4e14-85e6-1035b84483af.gif)